bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,259 @@ class CoffeeLintBear:
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=80,
+                        apply_max_line_length_on_comments: bool=True,
+                        space_before_and_after_arrow: bool=False,
+                        braces_spacing: bool=False,
+                        braces_spacing_width: int=0,
+                        spacing_in_empty_braces: int=0,
+                        camelcase_classes_name: bool=True,
+                        spaces_before_and_after_colon: bool=True,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        end_file_with_a_single_newline: bool=False,
+                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                        no_tabs: bool=True,
+                        newlines_after_classes: bool=True,
+                        number_of_lines_after_classes: int=3,
+                        no_backticks: bool=True,
+                        no_implicit_braces: bool=True,
+                        no_function_call_without_parentheses: bool=True,
+                        no_interpolation_in_single_quotes: bool=True,
+                        no_stand_alone_at: bool=True,
+                        no_throwing_strings: bool=True,
+                        no_trailing_semicolons: bool=True,
+                        no_trailing_whitespaces: bool=True,
+                        apply_no_trailing_whitespaces_in_comments: bool=True,
+                        apply_no_trailing_whitespaces_in_empty_line: bool=True,
+                        no_unnecessary_double_quotes: bool=True,
+                        use_english_operator: bool=False,
+                        space_around_operators: bool=True,
+                        space_after_comma: bool=True,
+                        cyclomatic_complexity: bool=True,
+                        cyclomatic_complexity_value: int=10,
+                        duplicate_keys: bool=True,
+                        enforce_parentheses_on_empty_constructor: bool=True,
+                        line_endings: bool=False,
+                        line_endings_option: str='unix',
+                        no_this: bool=False,
+                        no_plusplus: bool=False,
+                        no_empty_parameter_list: bool=False,
+                        no_empty_functions: bool=False):
+        """
+        :param: max_line_length:
+            Maximum number of characters per line.
+        :param: apply_max_line_length_on_comments:
+            Determines if ``max_line_length`` should also affects comments or
+            not.
+        :param: space_before_and_after_arrow:
+            Determines if spaces should be used before and/or after the arrow.
+        :param: braces_spacing:
```

Comment:
maybe ``check_braces_spacing`` to specify that it checks for it and you dont have to give it a number or whatever?

might be only me, don't make the change if you dont believe it should be there